app/src/views/camera/CameraContainer.tsx

```diff
@@ -28,57 +22,65 @@ export default function Camera({ navigation, firestoreCtrl, route }: any) {
     requestPermission,
     camera,
     picture,
+    description,
     isCameraEnabled,
     isFlashEnabled,
-    zoom,
+    isLocationEnabled,
     toggleCameraFacing,
     toggleFlashMode,
+    toggleLocation,
+    toggleCameraState,
+    setDescription,
     takePicture,
-    imageUrlGen,
-    setIsCameraEnabled,
+    makeChallenge,
+    goBack,
   } = useCameraViewModel(firestoreCtrl, navigation, route);
 
   if (!permission) {
     return (
-      <View style={styles.container}>
+      <ThemedView style={styles.container}>
         <Text style={styles.message}>
           Errors occurred while requesting permission
         </Text>
         <Button onPress={requestPermission} title="Grant Permission" />
-      </View>
+      </ThemedView>
     );
   }
 
   if (!permission.granted) {
     return (
-      <View style={styles.container}>
+      <ThemedView style={styles.container}>
         <Text style={styles.message}>
           We need your permission to show the camera
         </Text>
         <Button onPress={requestPermission} title="Grant Permission" />
-      </View>
+      </ThemedView>
     );
   }
 
   return (
-    <View style={styles.container}>
+    <ThemedView style={styles.container} colorType="backgroundPrimary">
+      <TopBar title="Camera" leftIcon="chevron-down" leftAction={goBack} />
       {isCameraEnabled ? (
-        <CameraView
-          style={styles.camera}
-          facing={facing}
-          enableTorch={isFlashEnabled}
-          ref={camera}
-          zoom={zoom}
-          testID="camera-view"
-        >
-          <View style={styles.buttonPlaceHolder}>
+        <ThemedView style={styles.cameraContainer} colorType="transparent">
+          <CameraView
+            style={styles.camera}
+            facing={facing}
+            enableTorch={isFlashEnabled}
+            ref={camera}
+            zoom={0}
```

Comment:
We should remove the zoom right ?

Reply:
Yes this just sets it to 0 at the beginning, because it could be to another value set by precedent actions from the user.